NEWS

```diff
@@ -192,6 +198,10 @@ Changed Functionality
   Previously, ``network_time()`` was used. This matters if ``Broker::publish()``
   is called within scheduled events or called within remote events.
 
+- The SSL analyzer now reports the correct version of SSLv2 connections as
+  passed during the hello messages. Zeek previously reported it as v2 always,
+  no matter was was passed.
```

Comment:
I would phrase this slightly different. How about

The SSL analyzer now reports the correct version when an SSLv2 client hello is used. Zeek previously always reported these as v2, even when the v2 client hello indicated support for a later version of SSL.

I know it's splitting hairs a bit, but the semantics are quite important :)

Reply:
Done, thanks